conans/client/loader_parse.py

```diff
@@ -12,14 +12,6 @@
 from conans.model import Generator
 
 
-def load_conanfile_class(conanfile_path):
-    loaded, filename = _parse_file(conanfile_path)
-    try:
-        return _parse_module(loaded, filename)
-    except Exception as e:  # re-raise with file name
-        raise ConanException("%s: %s" % (conanfile_path, str(e)))
-
-
 def _parse_module(conanfile_module, filename):
```

Comment:
Could be moved to the loader module and remove the loader_parse.py? or it still makes sense to keep is isolated? (Probably only used in the load class and import with "_" underscore named function)